frontend/hooks/forum/posts/delete/use-delete-post.ts

```diff
@@ -0,0 +1,25 @@
+import { useTranslations } from "next-intl";
+import { toast } from "sonner";
+
+import { mutationApi } from "./mutation-api";
+
+interface Props {
+  id: number;
+}
+
+export const useDeletePost = ({ id }: Props) => {
+  const tCore = useTranslations("core");
+
+  const deletePost = async () => {
+    const mutation = await mutationApi({ id });
+
+    if (mutation.error) {
+      let error: Error = mutation.error as Error;
+      toast.error(tCore("errors.title"), {
+        description: error.message
+      });
+    }
```

Comment:
```suggestion
    if (mutation.error) {
      toast.error(tCore("errors.title"), {
        description: tCore("errors.internal_server_error")
      });
    }
```

We don't want to show full error message from backend for basic user. It makes our users to confused.